src/Init/Notation.lean

```diff
@@ -22,6 +22,8 @@ def term : Category := {}
 def tactic : Category := {}
 /-- The syntax category for elements that can appear in the `do` notation. -/
 def doElem : Category := {}
+/-- The level syntax category. Declarations can be annotated with attributes using the `@[...]` notation. -/
```

Comment:
Could you please remove `Declarations can be annotated with attributes using the `@[...]` notation.` ?
It seems to be a copy&paste leftover.

Reply:
this comment was completely rewritten in the other commit anyway